platform/pulpcore/app/apps.py

```diff
@@ -38,13 +55,38 @@ def __init__(self, app_name, app_module):
 
         # Mapping of model names to viewsets (viewsets unrelated to models are excluded)
         self.named_viewsets = None
+        # Mapping of serializer names to serializers
+        self.named_serializers = None
 
     def ready(self):
         # register signals here as suggested in Django docs
         # https://docs.djangoproject.com/en/1.8/topics/signals/#connecting-receiver-functions
         import pulpcore.app.signals  # noqa
 
         self.import_viewsets()
+        self.import_serializers()
+
+    def import_serializers(self):
```

Comment:
It would be nice if this could be combined with `import_viewsets` since the code is nearly identical.